fedora_distro_aliases/cli.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+import sys
+
+from fedora_distro_aliases import get_distro_aliases
+
+GETTERS = {
+    "namever": lambda x: x.namever,
+    "branch": lambda x: x.branch,
+    "version": lambda x: x.version,
+}
+
+
+def run(output_type: str, aliases: list[str]):
+    getter = GETTERS[output_type]
+    distro_aliases = get_distro_aliases()
+
+    resolved_aliases: set[str] = set()
+    for alias in aliases:
+        resolved_aliases.update(getter(x) for x in distro_aliases[alias])
```

Comment:
The `GETTERS` seems redundant to me because you can do something like this here:

```python
new = {getattr(x, output_type) for x in distro_aliases[alias]}
resolved_aliases.update(new)
```

but I don't mind either option.